packages/ocapn/test/_table.js

```diff
@@ -0,0 +1,523 @@
+const sym = s => `${s.length}'${s}`;
+const str = s => `${s.length}"${s}`;
+const bts = s => `${s.length}:${s}`;
```

Comment:
Consider `assert(s.every(b => b <= 0xff))`

Reply:
this is redundant with https://github.com/endojs/endo/pull/2766#discussion_r2047425192 correct?

Reply:
determined correct